tests/test_command_to_gcode.py

```diff
@@ -0,0 +1,277 @@
+import unittest
+from unittest import mock
+import asyncio
+
+from apollo.command_to_gcode import CommandToGCode
+
+class CommandToGCodeTestCase(unittest.TestCase):
+
+
+    def setUp(self):
+
+        self.mock = mock.Mock()
```

Comment:
Based on the code below, I think this could be `self.send_mock` instead of just `self.mock`